src/declarations/stencil-private.ts

```diff
@@ -1145,7 +1145,7 @@ export interface DevServerContext {
   getBuildResults: () => Promise<CompilerBuildResults>;
   getCompilerRequest: (path: string) => Promise<CompilerRequestResponse>;
   isServerListening: boolean;
-  logRequest: (req: { method: string; pathname?: string }, status: number) => void;
+  logRequest: (req: HttpRequest, status: number) => void;
```

Comment:
I may be totally missing something here - I pulled down `main@HEAD` and enabled `strictFunctionTypes` (so any errors you fixed here wouldn't be on my branch and I would see the causers). I don't see a cause/error related to this change though. Can you point me to where/what we needed to update?

Reply:
I see this error:

```
[11:56:24 AM] Starting compilation in watch mode...

src/dev-server/server-context.ts:126:5 - error TS2322: Type '(req: d.HttpRequest, status: number) => void' is not assignable to type '(req: { method: string; pathname?: string; }, status: number) => void'.
  Types of parameters 'req' and 'req' are incompatible.
    Type '{ method: string; pathname?: string; }' is missing the following properties from type 'HttpRequest': acceptHeader, url, searchParams

126     logRequest,
        ~~~~~~~~~~

  src/declarations/stencil-private.ts:1149:5
    1149     logRequest: (req: { method: string; pathname?: string }, status: number) => void;
             ~~~~~~~~~~
    The expected type comes from property 'logRequest' which is declared here on type 'DevServerContext'

[11:56:30 AM] Found 1 error. Watching for file changes.
```

Reply:
I get that error both on `main` if I do

```sh
npx tsc --noEmit --strictFunctionTypes
```

and if I undo that single line on `ap/strict-function-types`

Reply:
Yuppppp classic PEBKAC. Thanks!